src/components/inputs/ColorSelect/ColorSelect.types.ts

```diff
@@ -71,4 +71,8 @@ export type Props = IrisInputProps<{
    * [default = 'bottom']
    */
   attach?: Attach | AttachAlias;
+  /**
+   * Shows hue slider, default = true
+   */
+  hueSlider?: boolean;
```

Comment:
Nit, but can we make this `showHueSlider`? I would expect `hueSlider` on its own to accept a JSX element.

Reply:
That's fair, I'll update.